lib/components/analysis/cards/cards.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:percent_indicator/percent_indicator.dart';
+
+
+class MyCards extends StatelessWidget {
```

Comment:
Please add comments for details the goal of the function and the parameters that it takes